edx-platform/pearson-theme/lms/templates/header/navbar-logo-header.html

```diff
@@ -14,7 +14,7 @@
 %>
 
 <h1 class="header-logo">
-  <a href="${branding_api.get_home_url()}">
+  <a hhref="${configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)}">
```

Comment:
Typo, hhref -> href @amalbas 
branding_api is not site aware?

Reply:
the get_home_url points to dashboard, for user not authenticated too so that is the problem: https://github.com/Pearson-Advance/edx-platform/blob/master/lms/djangoapps/branding/api.py#L611.